cli/admin/README.md

```diff
@@ -52,7 +52,7 @@ You can run the Pravega Admin CLI as follows:
 ```
 ./bin/pravega-admin
 ```
-You will se an output related to the default configuration parameters available at `conf/config.properties`
+You will se an output related to the default configuration parameters available at `conf/config.properties`. The values can be changed at the properties file, or use `config set property=...` to change the value.
```

Comment:
Fix existing typo: `se` -> `see`

Reply:
@dellThejas I still see the typo, have you resolved that locally? If so, you need to push the changes to resolve a comment.

Reply:
I see, I fixed it locally. I will push the changes and then resolve then.